src/contactForm.js

```diff
@@ -319,8 +322,8 @@ class RecrasContactForm {
                             field: this.findElement('[data-identifier="boeking.datum"]'),
                             i18n: RecrasCalendarHelper.i18n(this.languageHelper),
                             numberOfMonths: 1,
-                            toString(date, _format) {
-                                return RecrasDateHelper.datePartOnly(date);
+                            onSelect: (date) => {
+                                this.selectedDate = date;
```

Comment:
is there a way to test this, or at least document the expected types for inputs and results